examples/blocks-query.py

```diff
@@ -4,12 +4,19 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+"""
+The example demonstrates querying blocks of transactions - we can query blocks one by one
+after calling function:  Iroha.blocks_query
+"""
+
 from iroha import Iroha, IrohaGrpc
 from iroha import IrohaCrypto
 import os
 import sys
 import time
 import uuid
+import grpc  # grpc.RpcError
```

Comment:
I'll approve regardless, but if this is the only import, the `from grpc import RpcError` would have conveyed the same idea.

Reply:
Corrected: b36447fc3002b2d8b7617591a3363021126a443f